docs/en/glossary/glossary.asciidoc

```diff
@@ -185,6 +185,14 @@ local cluster. For more information, see {stack-ov}/xpack-ccr.html[{ccr-cap}].
 +
 //Source: X-Pack
 endif::xpack-terms[]
+ifdef::elasticsearch-terms[]
+[[glossary-ccs]] {ccs} (CCS)::
+
+The {ccs} feature enables any node to act as a federated client across
+multiple clusters. See {ref}/modules-cross-cluster-search.html[Cross-cluster search].   
```

Comment:
The hyphenation has changed recently, so I recommend using attributes for all CCS occurrences:
```suggestion
multiple clusters. See {ref}/modules-cross-cluster-search.html[{ccs-cap}].   
```

Reply:
Good catch! Pushed e948189 to update.